runtime/src/main/java/io/quarkiverse/githubaction/runtime/GitHubEvent.java

```diff
@@ -159,34 +159,27 @@ public DynamicGraphQLClient getDynamicGraphQLClient() {
     }
 
     public GHRepository getGHRepository() {
-        var repo = gitHubRepository;
-        if (repo == null) {
+        GHRepository localGitHubRepository = gitHubRepository;
```

Comment:
It's really personal taste but I'm not a big fan of `var`. I agree it's shorted by when you review stuff on GitHub and you don't have the full context of the IDE, it can be annoying to figure out the type.
In this case, it's relatively simple so I would have kept it but while I was adjusting micro things...